RELEASE_NOTES.md

```diff
@@ -11,6 +11,31 @@
 ## New Features
 
 - Package-based versioning has been introduced.
+- Rules for recurring dispatches have been added.
+  Examples:
+    Every 6 months:
+    message RecurrenceRule {
```

Comment:
Nitpicking: I'd suggest adding code in backticks, so that it looks nicely formatted :)